lib/extensions/polyglot_with_database/coursemology/polyglot/language.rb

```diff
@@ -0,0 +1,76 @@
+# This extends +Coursemology::Polyglot::Language+ to support integration into a database.
```

Comment:
@allenwq this is replaced with just one module definition, concrete languages are now defined in the gem, see Coursemology/polyglot#2